.gitignore

```diff
@@ -18,3 +18,4 @@ yarn-error.log
 /.fleet
 /.idea
 /.vscode
+composer.lock
```

Comment:
@Marshud I had reverted the last commit which removed the composer.lock this is a standard file which should not be removed as its needed by the deployment process

Reply:
Duly noted